src/AbstractSystems.fypp

```diff
@@ -0,0 +1,74 @@
+#:include "../include/common.fypp"
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES + CMPLX_KINDS_TYPES
+module LightKrylov_AbstractSystems
+    use LightKrylov_Constants
+    use LightKrylov_AbstractVectors
+    use LightKrylov_AbstractLinops
+    implicit none
+    private
+
+    character*128, parameter :: this_module = 'LightKrylov_AbstractSystems'
```

Comment:
Change to

```
character(len=128), parameter :: this_module = 'Lightkrylov_AbstractSystems'
```

to be consistent with the modern Fortran convention used in the other modules.

Reply:
done.